uint64 timestamp
float64[3] accl
float64[3] gyro
float64[2] odom
float64[2] gps1_pos
float64[2] gps1_vel
float64[2] gps2_pos
float64[2] gps2_vel
